addons/popochiu/editor/helpers/popochiu_gui_templates_helper.gd

```diff
@@ -65,7 +62,8 @@ static func copy_gui_template(template_name: String) -> void:
 	# Save the GUI template in Settings and popochiu_data.cfg --------------------------------------
 	_update_settings_and_config(template_name, commands_path)
 	
-	_progress_window.queue_free()
+	PopochiuUtils.print_normal("[wave]GUI template assignation finished[/wave]")
```

Comment:
```suggestion
	PopochiuUtils.print_normal("[wave]Selected GUI template successfully applied[/wave]")
```

"Assignation" has a very different meaning than in Latin languages (though it's almost the same in both Italian and Spanish).